<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiTestClass</class>
 <widget class="QMainWindow" name="uiTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UI测试</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>209</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1" rowspan="2" colspan="3">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="secondLabel">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>200</height>
           </size>
          </property>
          <property name="text">
           <string>辅轴波形</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="secondAxis">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>200</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>CrossCursor</cursorShape>
          </property>
          <property name="frameShape">
           <enum>QFrame::WinPanel</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="firstlabel">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>200</height>
           </size>
          </property>
          <property name="text">
           <string>主轴波形</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="firstAxis">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>200</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>CrossCursor</cursorShape>
          </property>
          <property name="frameShape">
           <enum>QFrame::WinPanel</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="comboBox_2">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>COM1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COM6</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="3" rowspan="4">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>168</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>断开</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_8">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>测试</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>71</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>自动模式</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>浏览</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="5" column="0" rowspan="2" colspan="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>主轴 幅值、频率</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>23</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Pi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Hz</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>辅轴 幅值、频率</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>工况编号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_6">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>23</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Pi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Hz</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="6" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="text">
         <string>预览</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>运行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>保存工况</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>comboBox_2</zorder>
   <zorder>horizontalSpacer</zorder>
   <zorder></zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>textBrowser</zorder>
   <zorder>lineEdit_6</zorder>
   <zorder>label_7</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openAction"/>
  </widget>
  <action name="openAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="uiTest.qrc">
     <normaloff>:/uiTest/Resources/ICon/folder.ico</normaloff>:/uiTest/Resources/ICon/folder.ico</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="icon">
    <iconset resource="uiTest.qrc">
     <normaloff>:/uiTest/Resources/ICon/folder.ico</normaloff>:/uiTest/Resources/ICon/folder.ico</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="uiTest.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>uiTestClass</receiver>
   <slot>slotControlInput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>uiTestClass</receiver>
   <slot>slotViewCurve()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>657</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>uiTestClass</receiver>
   <slot>slotSerialPortInit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>528</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>uiTestClass</receiver>
   <slot>slotSerialPortSendData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>652</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>629</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>uiTestClass</receiver>
   <slot>slotStopSys()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>653</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>566</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>uiTestClass</receiver>
   <slot>slotSerialPortClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>uiTestClass</receiver>
   <slot>slotSerialPortTestInit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>517</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>uiTestClass</receiver>
   <slot>slotSaveFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>649</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>645</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_2</sender>
   <signal>textChanged(QString)</signal>
   <receiver>uiTestClass</receiver>
   <slot>slotChangeFirstAxisF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>617</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>619</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>uiTestClass</receiver>
   <slot>slotSelectGKFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal1()</signal>
  <slot>slot1994()</slot>
  <slot>slotControlInput()</slot>
  <slot>slotViewCurve()</slot>
  <slot>slotTest2()</slot>
  <slot>slotSerialPortInit()</slot>
  <slot>slotSerialPortSendData()</slot>
  <slot>slotSerialPortReciveData()</slot>
  <slot>slotSerialPortClose()</slot>
  <slot>slotSerialPortTestInit()</slot>
  <slot>slotSaveFile()</slot>
  <slot>slotChangeFirstAxisA()</slot>
  <slot>slotChangeGKPram()</slot>
  <slot>slotSelectGKFile()</slot>
  <slot>slotSelectGKNumber()</slot>
  <slot>slotStopSys()</slot>
 </slots>
</ui>
